tar.go

```diff
@@ -29,11 +30,13 @@ func NewTarStreamer(r io.Reader, keywords []string) Streamer {
 		tarReader:  tar.NewReader(pR),
 		keywords:   keywords,
 	}
-	go ts.readHeaders() // I don't like this
+
+	go ts.readHeaders()
```

Comment:
Also, for #48, I'm not sure how to handle comparing two manifests where they have both `time` and `tar_mtime` defined.